source/_config.yml

```diff
@@ -58,6 +58,7 @@ symlink_latest:
   - facter
   - pe
 lock_latest:
+  - puppetdb 4.1
```

Comment:
Looks like maybe I swap the [dash for a colon](https://github.com/puppetlabs/puppet-docs/pull/625/files#diff-e6422af2c5a3ea680839278540cfdf29L58)? I'll make the change.